listenbrainz/db/tests/test_dump.py

```diff
@@ -47,6 +47,7 @@ def tearDown(self):
         shutil.rmtree(self.tempdir)
 
     def test_create_private_dump(self):
+        assert False
```

Comment:
oops

Reply:
I added this intentionally to ensure the fix works. Will remove this before merging.